be/src/common/config.h

```diff
@@ -95,11 +95,15 @@ CONF_Int32(push_worker_count_high_priority, "3");
 
 // The count of thread to publish version per transaction
 CONF_mInt32(transaction_publish_version_worker_count, "0");
+// The min count of thread to publish version per transaction
+CONF_Int32(transaction_publish_version_worker_count_min, "0");
 
 // The count of thread to apply rowset in primary key table
 // 0 means apply worker count is equal to cpu core count
 CONF_mInt32(transaction_apply_worker_count, "0");
 CONF_mInt32(get_pindex_worker_count, "0");
+CONF_Int32(transaction_apply_worker_count_min, "0");
```

Comment:
similar question, how does this configvar differs to `transaction_apply_worker_count`, and when `transaction_apply_worker_count` set to `0`, it means use the number of cores, will it apply to `transaction_apply_worker_count_min` as well?

make it dynamic adjustable if possible.

Reply:
I'll adjust the change attribute to dynamically update